example/lib/main.dart

```diff
@@ -70,40 +67,16 @@ class Storyboard extends StatelessWidget {
                     );
                   },
                 ),
-                ComponentState(
-                  docs: '''
-                    IconButton(onPressed: () {}, icon: Icon())
-                  ''',
-                  stateName: 'Secondary',
-                  builder: (context, c) {
-                    return Center(
-                        child: IconButton(
-                      icon: Icon(Icons.smartphone),
-                      onPressed: () {},
-                    ));
-                  },
-                ),
               ],
             ),
             Component(
               componentName: 'List',
               states: [
                 ComponentState(
+                  docName: 'ListExample',
```

Comment:
this seems like it should be something like `componentClassName` or something similar